src/app/ReadClient.h

```diff
@@ -323,6 +323,17 @@ class ReadClient : public Messaging::ExchangeDelegate
      */
     CHIP_ERROR SendRequest(ReadPrepareParams & aReadPrepareParams);
 
+    /**
+     *  Re-activate an idle subscription.
+     *
+     *  When subscribing to ICD and liveness timeout reached, the read client will move to the `InactiveICDSubscription` state and
+     * resubscription can be triggered via OnActiveModeNotification().
```

Comment:
That's the default policy, not guaranteed.  Replace "will" with "can"?

Reply:
As I think we should setting the state before calling `OnResubscriptionNeeded`, will should still be used here.